<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MusLIfe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #49423d;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">background-color: #737373;</string>
     </property>
     <attribute name="title">
      <string>                            ПОИСК МУЗЫКИ               </string>
     </attribute>
     <widget class="QLineEdit" name="searchingInput">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>741</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #CFCFCF;
font-size: 20px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="searchingButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>50</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;
color: rgb(255, 248, 235);
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dowload">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>640</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;
color: rgb(255, 248, 235);
font-size: 17px;</string>
      </property>
      <property name="text">
       <string>СКАЧАТЬ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addPlaylist">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>640</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;
color: rgb(255, 248, 235);
font-size: 15px;</string>
      </property>
      <property name="text">
       <string>Добавить в плейлист</string>
      </property>
     </widget>
     <widget class="QLabel" name="textPlaylist">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>640</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 15px;
color: rgb(255, 248, 235);
</string>
      </property>
      <property name="text">
       <string>Введите название плейлиста:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>640</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #CFCFCF;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="musicList">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>911</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #CFCFCF;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="messages">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>710</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="styleSheet">
      <string notr="true">background-color: #737373;</string>
     </property>
     <attribute name="title">
      <string>                          Проигрывание музыки                  </string>
     </attribute>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>660</y>
        <width>90</width>
        <height>90</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="right">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>680</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="left">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>680</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imageMusic">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>94</y>
        <width>861</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>                                 -</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playPlaylist">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;
color: rgb(255, 248, 235);
font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Проиграть плейлист</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playMusic">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;
color: rgb(255, 248, 235);
font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Проиграть песню</string>
      </property>
     </widget>
     <widget class="QSlider" name="verticalSlider">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>90</y>
        <width>22</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="rewindButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>670</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true">background-color: #737373;</string>
     </property>
     <attribute name="title">
      <string>                                Плейлисты                                    </string>
     </attribute>
     <widget class="QComboBox" name="playLists">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>941</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 5px solid rgb(77, 77, 77);
border-radius: 10px;
background: #ABABAB;
font-size: 35px;
color: #393939;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createPlaylist">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>311</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;
color: rgb(255, 248, 235);
font-size: 30px;</string>
      </property>
      <property name="text">
       <string>Создать плейлист</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletePlaylist">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>260</y>
        <width>281</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;
color: rgb(255, 248, 235);
font-size: 30px;</string>
      </property>
      <property name="text">
       <string>Удалить Плейлист</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openPlaylist">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>260</y>
        <width>251</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
border-radius: 10px;
color: rgb(255, 248, 235);
font-size: 26px;</string>
      </property>
      <property name="text">
       <string>Открыть Плейлист</string>
      </property>
     </widget>
     <widget class="QLabel" name="choiceText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
color: rgb(255, 248, 235);
background-color: #9B9B9B;
border-radius: 12px;</string>
      </property>
      <property name="text">
       <string>Выбирите Плейлист</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>420</y>
        <width>741</width>
        <height>321</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
